pyccl/core.py

```diff
@@ -68,6 +69,12 @@
 }
 
 
+class _Defaults:
+    """Default cosmological parameters used throughout the library."""
+    T_CMB = 2.725
```

Comment:
Note that we want to make this 2.7255 in v3. Making it 2.725 was one of the biggest blunders at the start of CCL

Reply:
Open an issue, and set it as a v3 milestone, or we'll forget! We'll change `Neff` too, to the newest value.

Reply:
OK, but I'm saying, should we add a deprecation warning here when someone relies on the default value?

Reply:
A deprecation warning for something so central that is used almost everywhere in CCL and is piped to many other functions? That's a tough one, because `_Defaults.T_CMB` gets called on import multiple times, as it is the default argument of many functions. Instead of tapping into `_Defaults.__getattr__` to warn, I think the best plan here is to warn one single time, on import, and that's it. The warning message will be thrown from `pyccl/__init__.py`.

Reply:
Sounds good to me. It'll be a bit annoying, but worth it.